Doc/library/zipfile.rst

```diff
@@ -79,6 +79,12 @@ The module defines the following items:
    :ref:`zipinfo-objects`.
 
 
+.. warning::
+
+   A leading slash in the filename may lead to the archive being impossible to
+   open in some zip programs on Windows systems.
+
+
```

Comment:
I'm not sure this needs to be warnings (red boxes), rather than just mentioned in the text.

If warning then just one and not three (I'm not sure where).